Webspage 3.0/graphics.htm

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="styles/main.css">
+</head>
```

Comment:
You may also want to add a <title> tag in the head. This is what shows up in the tab/browser.